File: 6-day/calculator.js
function calculator(number1, number2, operator) {
  switch (operator) {
    case '+':
      return number1 + number2
    case '-':
      return number1 - number2
    case '*':
      return number1 * number2
    case '/':
      return number1 / number2
    default:
      return 'Error'
  }
}

let number1 = parseFloat(prompt('Enter a number 1: '))
let number2 = parseFloat(prompt('Enter a number 2: '))
let operator = prompt('Enter operator: ')

let result = calculator(number1, number2, operator)
let outputMessage = result != 'Error' ? `${number1} ${operator} ${number2} = ${result}` : `There is an error`

alert(outputMessage)

File: 6-day/calculator.py
def calculator(number1, number2, operator):
    if (number2 == 0 and operator == '/'):
        return 'Error'
    else:
        operators = {
            '+': number1 + number2,
            '-': number1 + number2,
            '*': number1 + number2,
            '/': number1 + number2,
        }
        return operators.get(operator, 'Error')

number1 = float(input('Enter number 1: '))
number2 = float(input('Enter number 2: '))
operator = input('Enter the operator: ')

result = calculator(number1, number2, operator)
if (result == 'Error'):
    print('There is an error')
else:
    print('{} {} {} = {}'.format(number1, operator, number2, result))